test/atomist.config.ts

```diff
@@ -45,7 +45,7 @@ class StartUpListener extends AutomationEventListenerSupport {
 export const configuration: Configuration = {
     name: "@atomist/automation-node-tests",
     version: "0.0.6",
-    teamIds: [ "T1L0VDKJP" ],
+    teamIds: "T01234567",
```

Comment:
Shouldn't this still be an array?

Reply:
it works either way, but yeah, it might as well be